### DISCUSSION TITLE
What do we do with node commands

### DISCUSSION THREAD

**Discussion by kistenklaus** [mu-zero-HYPERLOOP/control-panel#44] [Announcements]
Right now at least that what i believe all commands have a seperate page, but iam sure if we have enough data to display on that page and we don't want a page, where there are just 2 buttons.

Maybe we can put the commands on the node page. What do you think about that?
@mu-zero-HYPERLOOP/siv

**Comment by NiklasHeneka** [comment]
Right now we don't even use pages for commands as the sidebar only shows object entries.
But I think having such sparse pages is not nice so we can put them in the node page.
